pcdsdevices/cvmi_motion.py

```diff
@@ -41,6 +46,15 @@ class CVMI(BaseInterface, GroupDevice):
 
     sample_paddle = Cpt(BeckhoffAxis, ':MMS:07', kind='normal')
 
+    # LEDs
+    led1_name = Cpt(EpicsSignal, ':LED:01:NAME', kind='normal')
+    led1_pct = Cpt(EpicsSignal, ':LED:01:ILL:PCT', kind='normal')
```

Comment:
```suggestion
    led1_pct = Cpt(PytmcSignal, ':LED:01:ILL:PCT', kind='normal', io="io")
```

All of these should be using `PytmcSignal`: https://confluence.slac.stanford.edu/display/PCDS/How+to+add+a+device+to+pcdsdevices#Howtoaddadevicetopcdsdevices-UsePytmcSignal

This is because pytmc generates two PVs for us in the case of an `io` variable:
`PCT` (what the user writes to to request a change) and `PCT_RBV` (what the PLC says the readback value is, usually reflecting what the user requested)

Reply:
Should all of the signals be type Pytmc in addition to pct?